docs/migration/2.0.0-migration.md

```diff
@@ -0,0 +1,101 @@
+# Bitops Core
+
+## Environment Variables
+Bitops is no longer using the `ENVIRONMENT` value, it instead uses; `BITOPS_ENVIRONMENT`
+
+Easiest migration solution; 
+Update startup command to read as followed;
+```
+docker run \
+-e BITOPS_ENVIRONMENT=$ENVIRONMENT   \    <-- If the ENVIRONMENT variable is being used. This is a simple approach. 
+-v /path/to/ops_repo:/opt/bitops_deployment    \
+bitovi/bitops:latest
+```
+
+### New Prefixes
+BitOps core exported environment variables now have a prefix of `BITOPS_`
+
+BitOps plugins (tool specific) environment variable exports are prefixed with; `BITOPS_<PLUGIN NAME>`
```

Comment:
What if I am using a custom plugin like `plugin-name-1`. Would it be `BITOPS_plugin-name-1` or `BITOPS_PLUGIN-NAME-1`?